examples/manifest.yml

```diff
@@ -0,0 +1,133 @@
+# `landing` expects this file to have the following format:
+# root {
+#   prelude: string
+#   children: [
+#     {
+#       # the path in the url, e.g. `rerun.io/examples/minimal`
+#       name: string
+#       # path to the python example
+#       python: string?
+#       # path to the rust example
+#       rust: string?
+#       # at least one of `python` or `rust` should be set
+#     }
+#     OR
+#     {
+#       # the path in the url, e.g. `rerun.io/examples/artificial-data`
+#       name: string
+#       # the title in the side nav + header, e.g. `Examples with Artificial Data`
+#       title: string
+#       # the content that's placed right after the header.
+#       # used to include setup instructions and general info
+#       prelude: string
+#       children: [
+#         { # same as above
+#           name: string
+#           python: string?
+#           rust: string?
+#         }
+#       ]
+#     }
+#   ]
+# }
+
+root:
+  prelude: |
+    In the Rerun [GitHub](https://github.com/rerun-io/rerun) repository we maintain
+    a list of examples that demonstrate using the Rerun logging APIs. Generally the
+    examples are individually self-contained, and can be run directly from a Git
+    clone of the repository. Many of the Python examples need additional dependencies
+    set up in a `requirements.txt` next to the example. These are noted in the
+    individual example sections below.
+
+    ## Setup
+
+    Make sure you have the Rerun repository checked out and the latest SDK installed.
+
+    ```bash
+    pip install --upgrade rerun-sdk  # install the latest Rerun SDK
+    git clone git@github.com:rerun-io/rerun.git  # Clone the repository
+    cd rerun
+    git checkout latest  # Check out the commit matching the latest SDK release
+    ```
+    > Note: Make sure your SDK version matches the examples.
+    For example, if your SDK version is `0.3.1`, check out the matching tag
+    in the Rerun repository by running `git checkout v0.3.1`.
```

Comment:
I think this might make sense as a stand alone page as well. Is it possible to create pages with things like setup instructions that are not connected to an example?

Reply:
Not with the current setup, I can do this part as a follow-up